LICENSE

```diff
@@ -7,3 +9,25 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+-------------------------ACKNOWLEDGEMENTS--------------------------------------
```

Comment:
Let's just enumerate them by class, leaves little unnecesasry room for interpretation:

```
Third-party Licenses

## com.whatever.classpath.to.DiskLruCache

<full license statement>

## com.whatever.classpath.to.PagerTabStrip

<full license statement>

## com.whatever.classpath.to.PagerTitleStrip

<full license statement>

```